<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgInputRestartYears</class>
 <widget class="QDialog" name="dlgInputRestartYears">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The First and Last Year of Restart Run</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>97</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFirstYear">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>52</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The first year of restart run</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLastYear">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>52</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The last year of restart run</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>29</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>First Year:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>29</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Year:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEndYear">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>53</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>29</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>End Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>29</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Base Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBaseYear">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>53</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgInputRestartYears</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgInputRestartYears</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
